content/300-guides/100-performance-and-optimization/090-metrics.mdx

```diff
@@ -0,0 +1,335 @@
+---
+title: 'Metrics'
+metaTitle: 'Metrics'
+metaDescription: 'Diagnose application performance with insights into Prisma Client database activity'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma metrics give you a detailed insight into how Prisma Client interacts with your database. You can use this insight to help diagnose performance issues with your application. If you send the data to an external metrics system, then you can view the data over time. For example, you might visualize how your application's number of idle and active connections change.
+
+</TopBlock>
+
+## About metrics
+
+Metrics provide information on the following aspects of Prisma Client:
+
+- The number of Prisma operations
+
+  A Prisma operation is a statement like `let user = prisma.user(include: {posts: true}})`, which Prisma Client translates into one or more SQL queries.
+
+- Query response times
```

Comment:
I'm not quite following what this section is about? You only mention some of the metrics/aspects. What is your aim here?

Reply:
OK, I've removed the section, because we cover the metrics in detail slightly below it anyway.